src/main/java/com/theironyard/services/BottleRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.theironyard.services;
+
+import com.theironyard.entities.Bottle;
+import com.theironyard.entities.User;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+
+/**
+ * Created by vasantia on 7/21/16.
+ */
+public interface BottleRepository extends JpaRepository<Bottle, Integer> {
+    List<Bottle> findByUser(User user);
+    Bottle findById(int id);
```

Comment:
This is redundant as the repository you are extending already has a findOne in it that does the same thing.